plugins/plugin-maven/che-plugin-maven-server/src/main/java/org/eclipse/che/plugin/maven/server/rest/MavenServerService.java

```diff
@@ -138,6 +149,15 @@ public String downloadSource(@QueryParam("projectpath") String projectPath, @Que
         return Boolean.toString(classpathManager.downloadSources(projectPath, fqn));
     }
 
+    @PUT
+    @Path("reimport/dependencies")
```

Comment:
It's more "noun" that "verb"

so it should be more an action on the noun "dependencies" like calling PUT on "/dependencies", no need to use a verb